.npmignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
You don't need to include the node_modules line in npmignore since this is automatically ignored - you can literally have an empty file for .npmignore and it will work the way you'd like : )